sections/03_categorize.md

```diff
@@ -139,66 +123,45 @@ setting, and model fine-tuning needed is very problem and dataset-specific.
 Moreover, this type of tasks often depends on both lesion localization and
 appearance that pose challenges for CNN-based approaches. Straightforward
 attempts to capture useful information from full-size images in all three
-dimensions simultaneously make applications of standard neural network
-architectures computationally unfeasible due to the curse of dimensionality.
-Instead, two dimensional models are often used to either process image slices
+dimensions simultaneously via standard neural network architectures were computationally unfeasible.
+Instead, two dimensional models were used to either process image slices
```

Comment:
I don't think 2.5D was entirely clear in this section, but I'm okay leaving it as is.  In terms of the phrasing, we've used 1D and one-dimensional heterogeneously elsewhere.  Maybe something to standardize.